tests/integration/scale_test.go

```diff
@@ -14,7 +14,7 @@ func scaleTests(t *testing.T, when spec.G, it spec.S) {
 	var service testutil.TestService
 
 	it.Before(func() {
-		service.Create(t, "nginx")
+		service.Create(t, "sangeetha/mytestcontainer")
```

Comment:
We should avoid using personal containers in our tests, if someone leaves and deletes their stuff all our tests break. Let's see if we can add image rancher account, or set it up as a repo in rancher account and create it via build.

Reply:
Agree, I'm using it here because it's the same image used for validations in rancher/rancher

Reply:
ok